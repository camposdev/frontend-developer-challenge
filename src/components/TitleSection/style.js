import styled from "styled-components";

export const TitleComponent = styled.h2`
  font-size: 1.8rem;
  text-align: center;
  margin-bottom: 50px;
`;