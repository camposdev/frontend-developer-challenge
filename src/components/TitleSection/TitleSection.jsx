import React from "react";
import PropTypes from "prop-types";
import { TitleComponent } from "./style";

const TitleSection = ({ children }) => (<TitleComponent>{children}</TitleComponent>);

TitleSection.propTypes = {
  children: PropTypes.any.isRequired
};

export default TitleSection;
