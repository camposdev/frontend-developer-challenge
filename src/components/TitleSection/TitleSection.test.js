import React from "react";
import renderer from "react-test-renderer";
import TitleSection from "./TitleSection";

describe("TitleSection test suit", () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<TitleSection>Helo World</TitleSection>)
      .toJSON();
    
    expect(tree).toMatchSnapshot();
  });
});