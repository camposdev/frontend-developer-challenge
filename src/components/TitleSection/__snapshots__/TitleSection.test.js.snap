// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleSection test suit renders correctly 1`] = `
<h2
  className="sc-bdVaJa hVrpyH"
>
  Helo World
</h2>
`;
